### DISCUSSION TITLE
Port Marco to Wayland

### DISCUSSION THREAD

**Discussion by ParkerGit2328** [mate-desktop/mate-wayland-session#19] [Ideas]
I think you should port the Marco window manager to Wayland.

**Comment by lukefromdc** [comment]
That is well beyond my capability, and would in truth be the application of the Marco name to a new, written from scratch
wlroots compositor. 

If someone with the skills wants to do it of course, I would test it. If someone makes this reality, 
a wayfire session could also still be shipped or a second, optional package depending on wayfire (the way compiz on 
GNOME 2 was optional) be shipped alongside such a session with our own compositor. Wayfire though is plugin based,
so the compiz-style eye candy can easily be turned off if desired. Main advantage would be not having to deal with the
GNOME gsettings values at all.  Contributing PR's to wayfire to solve this and similar MATE-specific problems by providing
options or plugins (e.g a MATE compatability plugin like for compiz) might actually be more useful.

Note that we are a small team, and whoever wrote MATE-specific compositor would need to commit to bottomlining
maintainance on that package.
